### DISCUSSION TITLE
lsb-release 切换到 lsb-release-minimal

### DISCUSSION THREAD

**Discussion by UTsweetyfish** [linuxdeepin/developer-center#3884] [Features Request & Ideas | 特性请求 & 头脑风暴]
Debian 独立维护了一个 lsb-release-minimal，实现了 lsb-release 的功能。

https://packages.debian.org/source/sid/lsb-release-minimal

**Comment by Zeno-sole** [comment]
lsb-release-minimal 好像是这个过度方案，切换过去有什么优势么？
